metadata-ingestion/src/datahub/ingestion/source/metadata/business_glossary.py

```diff
@@ -77,14 +85,22 @@ class BusinessGlossaryConfig(DefaultConfig):
     def version_must_be_1(cls, v):
         if v != "1":
             raise ValueError("Only version 1 is supported")
+        return v
+
+
+def create_id(path: List[str], enable_datahub_guid: bool) -> str:
```

Comment:
this should additionally take in the Optional `id`.

Reply:
Done